packages/users/src/users.ts

```diff
@@ -1,5 +1,6 @@
 import { Identity, PrivateKey } from '@textile/crypto';
 import _ from 'lodash';
+import 'websocket-polyfill';
```

Comment:
I added this to allow the authenticate flow which uses websocket to also work inside a node-env cc @studna

Reply:
actually I didn't include it on purpose in order to keep bundle size at minimum. It can be mentioned in doc (same as with Textile JS), that for nodejs you need to do:

```js
global.WebSocket = require('ws');
```

That polyfill does:
```js
if(isNode()) {
  global.WebSocket = require('websocket');
}
```

Tbh I don't know what's difference between `ws` and `websocket`, but `ws` seems more popular. 😅

Reply:
Okay. But how much larger does the websocket library add to the bundle size, we should measure that and see if it is significant. Because I believe it would be a greater experience if we support both envs out of the box.